create_db.js

```diff
@@ -9,13 +9,13 @@ pg.connect(conStringPri, (err, client, done) => { // eslint-disable-line no-unus
         process.exit(1);
     }
     // create the db and ignore any errors, for example if it already exists.
-    client.query(`CREATE DATABASE ${config.postgres.db};`, (err1) => { // eslint-disable-line no-unused-vars
+    client.query(`CREATE DATABASE "${config.postgres.db}";`, (err1) => { // eslint-disable-line no-unused-vars
```

Comment:
YMMV. :man_shrugging: 

Avoids the string interpolation with \`
`client.query('CREATE DATABASE "$1"', config.postgres.db, ...)`

Reply:
We took a look into this and it didn't work. :./